README.md

```diff
@@ -94,73 +138,103 @@ firmwares/
               *.ipk
 ```
 
-As you notice there are several different image variants ("backbone", "default", etc.).
-These different *packages lists* are defined in `packages/`.
-See the "Features" section above for a description of the purpose of each package list.
-With the "OpenWrt-Imagebuilder" you can assemble your own image variant with your
-*packages lists* without having to compile everything yourself. The "OpenWrt-SDK" is
-the fastest way to build your own packages or programs without compiling OpenWrt itself.
-The "initrd" directory contains some initrd-images for netboot, which are required on
-some boards to initially install OpenWrt.
+As you notice there are several different image variants ("manual", "notunnel",
+etc.). 
++ These different *packages lists* are defined in `packages/`. See the
```

Comment:
`packages/` --> `packagelists/`

Reply:
You talk here on the "firmware" folder, right? Then this packages folder is just the storage-repo of all packages available.These packages will be included according to the flavors in packagelists/ to the image or used as source then running opkg install on the router.